usdigitalresponse/grants-ingest
# Fix: Handle CFDA numbers with trailing plus signs in FFIS spreadsheets

### Ticket #261 

Fixes #261 

## Description

This PR implements a fix for the issue described in #261. It also adds a unit test for this scenario.

## Testing

1. Edit an FFIS spreadsheet so that one or more **CFDA number** cell values contain a trailing `+` character.
    - Hint: You can use the new fixture file added in this branch, `cmd/SplitFFISSpreadsheet/fixtures/example_spreadsheet_trailing_plus_sign_cfda.xlsx` for testing.
3. Submit the spreadsheet file to be parsed by the `SplitFFISSpreadsheet` Lambda function (ex. by uploading the spreadsheet to a key like `sources/YYYY/MM/DD/ffis/download.xlsx` in the source data bucket).
4. Check the parsed data saved to the prepared data bucket to ensure that they were parsed correctly.

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
